janus_core/src/test_util/kubernetes.rs

```diff
@@ -104,7 +104,7 @@ impl Cluster {
         .await
         .expect("timeout waiting for forwarded port to become writable");
 
-        PortForward { child }
+        PortForward { local_port, child }
```

Comment:
I think this is a pre-existing rustdoc error, but the doc comment for this method needs to be updated to refer to `PortForward`.